scripts/lighthouseRun.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e
 
-npx -p lighthouse@6 lighthouse http://localhost:7080/news/articles/cn7k01xp8kxo --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
-npx -p lighthouse@6 lighthouse http://localhost:7080/kyrgyz --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@11 lighthouse http://localhost:7080/news/articles/cn7k01xp8kxo --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
```

Comment:
Not quite related to this (but we might forget if we don't do it now).
 
I noticed that the scores have improved since upgrading to Lighthouse 11, and wondered if we should bump up the baseline scores? 

```
http://localhost:7080/news/articles/cn7k01xp8kxo
┌─────────┬─────────────────┬────────────┬─────────────┬────────┐
│ (index) │    category     │ scoreValue │ budgetValue │ status │
├─────────┼─────────────────┼────────────┼─────────────┼────────┤
│    0    │     'ally'      │    100     │     85      │ 'Pass' │
│    1    │ 'bestPractises' │    100     │     85      │ 'Pass' │
│    2    │      'seo'      │    100     │     100     │ 'Pass' │
└─────────┴─────────────────┴────────────┴─────────────┴────────┘
```

and
 
```
http://localhost:7080/kyrgyz
┌─────────┬─────────────────┬────────────┬─────────────┬────────┐
│ (index) │    category     │ scoreValue │ budgetValue │ status │
├─────────┼─────────────────┼────────────┼─────────────┼────────┤
│    0    │     'ally'      │    100     │     85      │ 'Pass' │
│    1    │ 'bestPractises' │    100     │     85      │ 'Pass' │
│    2    │      'seo'      │    100     │     100     │ 'Pass' │
└─────────┴─────────────────┴────────────┴─────────────┴────────┘
```

which means both pages are now returning 100 - could we update all budget values to 100? https://github.com/bbc/simorgh/blob/a175360d8c3146ee70d1c08930fb7b6fea540ebe/scripts/lighthouseBudget.js#L74-L78

Reply:
Not sure on the best approach tbh. 100 is the ideal to hit, but may become too constricting over time? 

Could bump it up to 90 maybe as a compromise?